README.md

```diff
@@ -111,11 +111,17 @@ The value of `test_args` can be accessed in `runtest.jl` via the `ARGS` variable
 using Test
 # ...
 
-if @isdefined(ARGS) && length(ARGS) > 0 && ARGS[1] == "only_fast_tests"
-    # run only fast tests
-    include("only_fast_tests.jl")
-else
-    # do something else
+# run fast tests by default
+include("fast_tests.jl")
+
+if @isdefined(ARGS) && length(ARGS) > 0 && ARGS[1] == "slow_tests"
```

Comment:
```suggestion
if @isdefined(ARGS) && "slow_tests" in ARGS
```
Maybe with an `in` test one could run both of the optional ones.

Also, I wonder if the `@isdefined` can be dropped. Is there a way to run Julia without having `ARGS` be defined?

Reply:
Github search for `@isdefined(ARGS)` has only <10 results, so probably we can go with just `if "slow_tests" in ARGS`